ChelseasFieldJournal/src/ItemJournal.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Linq;
+using System.Text;
+using Vintagestory.API.Client;
+using Vintagestory.API.Common.Entities;
+using Vintagestory.API.Common;
+using Vintagestory.API.MathTools;
+
+namespace ChelseasFieldJournal
+{
+    class ItemJournal : Item
+    {
+        public GuiDialogToolJournal dlg;
+        private ICoreClientAPI capi { get; set; }
+
+        public override void OnLoaded(ICoreAPI api)
+        {
+            base.OnLoaded(api);
+            this.capi = (api as ICoreClientAPI);
+        }
+        public override void OnHeldIdle(ItemSlot slot, EntityAgent byEntity)
+        {
+            base.OnHeldIdle(slot, byEntity);
+            if (this.api.Side == EnumAppSide.Client)
+            {
+                this.dlg = new GuiDialogToolJournal("gui-dialogs-journal", (EntityPlayer)byEntity, capi);
+                this.dlg.TryOpen();
+                this.dlg.OnClosed += delegate ()
+                {
+                    byEntity = null;
+                };
+            }
+        }
+        public override bool OnHeldInteractCancel(float secondsUsed, ItemSlot slot, EntityAgent byEntity, BlockSelection blockSel, EntitySelection entitySel, EnumItemUseCancelReason cancelReason)
+        {   
+            return true;
```

Comment:
We can simplify this. Remove the curly braces and use `public override bool OnHeldInteractCancel(/*...*/) => true`